src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react'
+import NavbarRB from 'react-bootstrap/Navbar'
+import Nav from 'react-bootstrap/Nav'
+import Form from 'react-bootstrap/Form'
+import FormControl from 'react-bootstrap/FormControl'
+import Button from 'react-bootstrap/Button'
+import NavDropdown from 'react-bootstrap/NavDropdown'
+
+interface Props {
+  /** Determines the navbar background color */
+  bg?: 'light' | 'dark'
+  /** Determines the letters color. It should be combined with the background color (bg) */
+  variant?: 'light' | 'dark'
+}
+
+/**
+ * Used to redirect users to the main topics.
+ */
+class Navbar extends Component<Props, {}> {
+  render() {
+    return (
+      <NavbarRB
+        bg={this.props.bg ? this.props.bg : 'dark'}
+        variant={this.props.variant ? this.props.variant : 'dark'}
+      >
+        <NavbarRB.Brand href="#">{' HospitalRun'}</NavbarRB.Brand>
+        <NavbarRB.Collapse id="responsive-navbar-nav">
+          <Nav className="mr-auto">
+            <Nav.Link href="#">Elem1</Nav.Link>
+            <Nav.Link href="#">Elem2</Nav.Link>
+            <Nav.Link href="#">Elem3</Nav.Link>
+            <NavDropdown title="Dropdown" id="collasible-nav-dropdown">
+              <NavDropdown.Item href="#action/3.1">Action1</NavDropdown.Item>
+              <NavDropdown.Item href="#action/3.2">Action2</NavDropdown.Item>
+              <NavDropdown.Item href="#action/3.3">Action3</NavDropdown.Item>
+            </NavDropdown>
+          </Nav>
+          <Nav>
+            <Form inline>
+              <FormControl type="text" placeholder="Search" className="mr-sm-2" />
+              <Button>Search</Button>
```

Comment:
We will need to allow for the frontend using this component to pass a click handler to this button. 

I think we should add a prop called `onSeachButtonClick` which will handle the click event.

Also, we can make use of the `Button` component that exists in this component library instead of the react-bootstrap library.